lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -2048,6 +2063,57 @@ pub struct CGOSharedDirectoryListRequest {
     pub path: *const c_char,
 }
 
+/// SyncLicenseStore protects an underlying LicenseStore with a mutex,
+/// making it safe for concurrent use.
+#[derive(Default)]
+struct SyncLicenseStore<L> {
+    inner: Mutex<L>,
+}
+
+impl<L> SyncLicenseStore<L> {
+    pub fn new(license_store: L) -> Self {
+        Self {
+            inner: Mutex::new(license_store),
+        }
+    }
+
+    #[allow(dead_code)]
+    pub fn into_inner(self) -> L {
+        self.inner.into_inner().unwrap()
+    }
```

Comment:
Do we need this?

Reply:
It might be dead code because the type itself is not public; being internal-use only I guess we can remove it and add it if and when we end up needing it?